.github/workflows/label_approved_pull_requests.yml

```
@@ -0,0 +1,19 @@
+name: Label approved pull requests
```

yamllint (1.35.1)

[error] wrong new line character: expected \n

(new-lines)